sqlserver-delta-plugins/src/e2e-test/features/mssql/DesignTimeValidation.feature

```diff
@@ -0,0 +1,138 @@
+#
+# Copyright © 2023 Cask Data, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+#
+
+@Mssql @Required
```

Comment:
Instead of marking the complete feature file required select specific scenarios and mark them as required.

Reply:
Added to specific scenarios